README.md

```diff
@@ -134,6 +134,9 @@ The Julia REPL is quite powerful.  See the section in the manual on
 [the Julia REPL](https://docs.julialang.org/en/v1/stdlib/REPL/)
 for more details.
 
+On Windows we highly recommend running Julia using a modern terminal,
+such as installing the [Windows Terminal from the Microsoft Store](https://aka.ms/terminal).
```

Comment:
Probably need some comment on how to use Julia from this terminal, since double-clicking `julia.exe` will open the console.

Reply:
We have more detailed instructions here https://julialang.org/downloads/platform/
Perhaps a little out of place to provide more details here? I think it would be better to add more details to the download page.

Reply:
Are you sure instructions are there? I think Steven is saying that installing Windows Terminal is not enough. How do you run Julia in Windows Terminal? And especially how to make this happen by simply clicking on `julia.exe` shortcut?

Reply:
Fair enough, we can add more instructions over at JuliaLang..org, but I don't think adding more than this is appropriate for the README

Reply:
Definitely should not be in the README. Ideally the installation instructions should be in the Julia documentation. For now, we should put them on the website - where they currently live.

Also, I feel we don't want to recommend Windows terminal, since it doesn't work on Windows 7, for example. Is there a generic way to describe what terminal is required and then list Windows terminal as one of the examples?

Reply:
@ViralBShah  Windows 7, is not even supported by Microsoft at this point, it was dropped at the start of the year, I don't see why we need to accommodate 100% functionality with users using an unsupported operating system from Microsoft itself. 

Windows 10 forces auto upgrades, as a result recommending the officially supported terminal sounds the most sensible thing here.

Reply:
Don't you need a recent Windows 10 from last year for Windows Terminal or is that fixed in the recent release? Julia currently does run on Windows 7! It can't hurt to add a single line (perhaps only on the Julia website) that a usable terminal should be provided.

If we want to bump the minimum Windows version we support to Windows 10, then let's get that decision taken.

Reply:
Yes you need a relatively recent version, but Windows 10 forces auto-updates on users, so it's not too important to worry about this case. 

Right, if a user is on Windows 7, they should find a better terminal and this doesn't change anything from the current scenario. This one-liner is strictly better than making no recommendation.

Reply:
@GregPlowman this is a fair point but that's actually an issue on the OS level that needs resolving and currently there is NO solution to this problem for any terminal on Windows. 

What we can do is create a `.bat`  called `Start Julia using the Windows Terminal` which could check if Windows Terminal exists and launch Julia from there if it exists. This would be a temporary solution, until OS level integration of the Windows Terminal is incorporated.

Reply:
We can merge this particular PR - but until we address the various things here, I don't think we should close #7267.

Reply:
@ViralBShah What exactly do we need to address to close that issue?  I want to put that issue out of its misery, since it hasn't been productive at all.

Reply:
I believe the following things would suffice to close it:

1. Instructions on how to use Windows Terminal, or even better, the .bat file solution that works automatically and falls back to the Console or some such thing.

2. In the installation instructions, some lines for those who don't have Windows Terminal. Many people have said in issues that their work machines are quite behind on Windows OS updates by years. So at least a clear message for those users that they can use other terminals. For those, we don't need detailed instructions or anything beyond the note.

Reply:
For 2, where should those instruction live?

Reply:
If using an alternative terminal with Julia is so complicated that we can't even document it clearly, that's not a good sign for the ability of new, unsophisticated users to be able to take our advice.

Reply:
I think the non-Terminal instructions should be literally what I do with my Windows 7 VM. Install git-bash, and start Julia with the full path.

Reply:
> If using an alternative terminal with Julia is so complicated that we can't even document it clearly, that's not a good sign for the ability of new, unsophisticated users to be able to take our advice.

It really isn't. The only issue is people on Windows 7, which is a 10 year old OS not even supported by Microsoft at this point, yet for some reason we need to accommodate them in the instructions.

Reply:
You don't need to do anything you don't like doing!

Reply:
Dropping support for something doesn't mean it suddenly stops working. I think it literally means

> We will not go out of our way to support these old things.